lib/models/samples_widget_list.dart

```diff
@@ -81,6 +82,7 @@ const sampleWidgets = <String, Widget>{
   'search_with_geocode': SearchWithGeocodeSample(),
   'select_features_in_feature_layer': SelectFeaturesInFeatureLayerSample(),
   'set_basemap': SetBasemapSample(),
+  'set_reference_scale': SetReferenceScaleSample(),
```

Comment:
v.next has just been updated with changes to how this class is defined. See if you can update your branch and match the new format here.

Reply:
Ok will do!